expression/builtin_info.go

```diff
@@ -245,6 +248,31 @@ func (b *builtinVersionSig) eval(_ []types.Datum) (d types.Datum, err error) {
 	return d, nil
 }
 
+type tidbVersionFunctionClass struct {
+	baseFunctionClass
+}
+
+func (c *tidbVersionFunctionClass) getFunction(args []Expression, ctx context.Context) (builtinFunc, error) {
+	tp := types.NewFieldType(mysql.TypeString)
+	tp.Flen = len(printer.GetTiDBInfo())
+	bf, err := newBaseBuiltinFuncWithTp(args, ctx, tpString)
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+	sig := &builtinTiDBVersionSig{baseStringBuiltinFunc{bf}}
```

Comment:
`tp` is not used.
set bf.tp.Flen = len(printer.GetTiDBInfo()) here,
and remove line 256 and line 257.

bf.tp.Tp has been set in newBaseBuiltinFuncWithTp.